config/prod.secret.exs

```diff
@@ -0,0 +1,41 @@
+# In this file, we load production configuration and secrets
```

Comment:
Normally this file should not be committed to the git repo. This must be removed from the template too 🙈

Reply:
Actually, in my understanding, it's not correct @olivierobert If we hard-code the value in this file then we need to add it to `.gitignore`

As here, I'm using it to load the `env` variable (following base Phonenix setup) also as my app is deploy to Heroku (don't Dockerize the app) so I need this file in order to build the app on Heroku @olivierobert 


https://hexdocs.pm/phoenix/deployment.html#handling-of-your-application-secrets

![Pasted_Image_7_14_20__17_22](https://user-images.githubusercontent.com/11751745/87414969-b7882100-c5f6-11ea-9843-112b4ffafea6.png)

Reply:
Heroku does support container deployments :-) Not needed for this specific project but using env variables + [Phoenix releases](https://hexdocs.pm/phoenix/releases.html) would be the best to go.

Reply:
Well if we design to go with https://hexdocs.pm/phoenix/releases.html then we could skip this file ;) 

I think we should add it to our `elixir-template` or Comppas that all Elixir projects should go with Release process, what do you think? so we won't have the conflict on each project 🌻

Reply:
That the goal with our Growth project 👉  https://trello.com/c/GN3EPBtO 😆